.envrc

```diff
@@ -1,11 +1,10 @@
 source activate structExpectations-analysis
 
 # We need the baseline directory quite frequently
-export PROJECT_DIR=$PWD
+export PYTHONPATH=$PWD:$PYTHONPATH
```

Comment:
Where is `PROJECT_DIR` defined now? It is used a couple of lines below. What is the motivation to change this here in the first place?